setup.py

```diff
@@ -103,7 +103,7 @@ def recursive_data_files(directory, install_directory):
     "craft-cli~=2.6",
     "craft-grammar>=2.0.1,<3.0.0",
     "craft-parts>=2.1.2,<3.0.0",
-    "craft-platforms~=0.1",
+    "craft-platforms~=0.4.0",
```

Comment:
This would require a manual update to 0.5.0, but I think (hope?) that the snapcraft build planner can be considered to have a stable API unless we need snapcraft-specific changes (in which case we'd be changing snapcraft in lockstep), so I think we can probably say:

```suggestion
    "craft-platforms~=0.4",
```

This would be equivalent to `>=0.4.0,<1.0.0` rather than `>=0.4.0,<0.5.0`.

Not a strong preference though, more a note to ensure we're doing what you intended.

Reply:
Yeah that should work.